src/HomeRunBall.purs

```diff
@@ -16,7 +16,7 @@ import Type.Row (Cons, Nil, kind RowList)
 -- | Check a string for validation rules provided by a row proxy and return a validation result
 checkRules :: forall row errors rl
```

Comment:
should be updated to take `a` and allow the String to be replaced by `a`?

Reply:
oh, sorry, realized you were talking about this specific thing. let me try to read more...

Reply:
btw this should be updated along with type VS becoming VV, https://github.com/justinwoo/purescript-home-run-ball/pull/5/files#diff-6577c4f181007b0640ffddc1d6f6f7fcR44